weblab/experiments/migrations/0032_experimentversion_experiment.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-02-27 12:11
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiments', '0031_remove_runnable_experiment'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
The models look like the right end state, but the migrations don't quite get us there reliably: I'm getting an error running them locally:
```
  Applying experiments.0032_experimentversion_experiment...Traceback (most recent call last):
  File "//anaconda3/envs/weblab/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
psycopg2.IntegrityError: insert or update on table "experiments_experimentversion" violates foreign key constraint "experiments_experime_experiment_id_5970cd0a_fk_experimen"
DETAIL:  Key (experiment_id)=(1) is not present in table "experiments_experiment".
```
Looks like it's the default value here that's causing issues, as I don't have an experiment with id 1 (it was deleted a while back...).

The issue I think is that you don't copy data from the `experiment_key` field to this new `experiment` field. If `experiment_key` was created as a proper `ForeignKey` initially, we could just have a `RenameField` migration here.

Reply:
So here we'd need
```
    operations = [
        migrations.RenameField(
            model_name='experimentversion',
            old_name='experiment_key',
            new_name='experiment',
        ),
        migrations.AlterField(
            model_name='experimentversion',
            name='experiment',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='versions', to='experiments.Experiment'),
        ),
    ]
```
```